ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/runs/[runID]/useRun.ts

```diff
@@ -0,0 +1,111 @@
+import { baseInitialFetchFailed } from '@inngest/components/types/fetch';
```

Comment:
It might be better to place this inside the `src/queries` folder, for consistency

Reply:
Do we want to put all fetching in there? Or only fetching that needs to be shared between components?

Reply:
Ignore me, I thought we only had inside the component vs query folder patterns, but the apps is already following this pattern. Use your best judgment, if this becomes a problem we can standardize.